src/lib.ts

```diff
@@ -88,6 +89,7 @@ export type BonusEquipMode = "free" | "embezzler" | "dmt" | "barf";
 export const WISH_VALUE = 50000;
 export const HIGHLIGHT = isDarkMode() ? "yellow" : "blue";
 export const ESTIMATED_OVERDRUNK_TURNS = 60;
+export const EMBEZZLER_MULTIPLIER = globalOptions.embezzlerMultiplier;
```

Comment:
The way that this is set up, passing it as an argument won't work. That's (probably) fine, but worth checking; EMBEZZLER_MULTIPLIER gets initialized before we actually run Args.fill

the easiest fix to this is to make it a () => number and make it get called everywhere it shows up. But we'd also have to see if those are top-level code or not.

The real answer here is 'that's fine, whatever, who is going to be passing this as an arg instead of using a pref.' Which is to say: I'm going to merge this and then fix it myself later maybe